src/Uno.UI/UI/Xaml/DragDropManager.cs

```diff
@@ -23,10 +23,16 @@ internal sealed class DragDropManager : CoreDragDropManager.IDragDropManager
 		public DragDropManager(Window window)
 		{
 			_window = window;
+
+#if __MACOS__
```

Comment:
I'm not sure how difficult it would be to get dependency injection working on macOS. Regardless, this should change when/if it is ever supported.

Reply:
Yes at some point it will be supported ... but for now this is good enough 👍